### PR TITLE
Software docs

### PR DESCRIPTION
**Pull request by aebruno**
Start to software docs.

### CODE REVIEW

File: pages/software/module-commands.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+specialized syntax.
+
+| Command                 | Description                         |
+| ----------------------- | ----------------------------------- |
+| `module avail`          | List available software.            |
+| `module spider <module>`| Searches for a particular software. |
+| `module load <module>`  | Load a module to use the software.  |
+| `module unload <module>`| Remove or unload a module           |
+| `module purge`          | Remove all modules.                 |
+| `module save <name>`    | Save the state of all loaded module |
```
Comment (dsajdak) [inline pages/software/module-commands.md:14]:
```suggestion
| `module save <name>`    | Save the state of all loaded modules |
```

File: pages/software/building.md [hunk 2]
```diff
@@ -1,5 +1,157 @@
...
+!!! Warning "You must build software on compile nodes"
+    You will not be able to compile software on the login nodes. You must first
+    ssh into a compile node before building software. After logging in to the
+    login node, simply run `ssh compile`.
+
+## Software build requests
+
+Users can request software be installed by CCR staff by submitting a new [issue
+on GitHub here](https://github.com/ubccr/software-layer/issues/new). Please
+provide the name of the software or library you're interested and any specific
```
Comment (dsajdak) [inline pages/software/building.md:19]:
```suggestion
provide the name of the software or library you're interested in and any specific
```

File: pages/software/building.md [hunk 3]
```diff
@@ -1,5 +1,157 @@
...
+CCR uses [EasyBuild](https://easybuild.io/) for building software. EasyBuild is
+a software build and installation framework that helps manage software on High
+Performance Computing (HPC) systems in an efficient and reproducible way.
+
+EasyBuild is written in python and uses simple human readable build recipes.
+EasyBuild provides a vast repository of build recipes for all kinds of software
+shared with the greater HPC community. There's a good chance that the software
+you're looking for already has an EasyBuild recipe.
+
+EasyBuild recipes, called easyconfigs, are text files (python syntax) used
```
Comment (dsajdak) [inline pages/software/building.md:38]:
```suggestion
EasyBuild recipes, called easyconfigs, are text files (python syntax) used to
```

File: pages/software/building.md [hunk 4]
```diff
@@ -1,5 +1,157 @@
...
+toolchain](releases.md) if necessary and build the software by running the
+following command:
+
+```
+# Build software
+$ eb SAMtools-X.X.X-GCC-Y.Y.Y.eb
+
+# Check what are the dependencies
+$ eb SAMtools-X.X.X-GCC-Y.Y.Y.eb -M
+
```
Comment (dsajdak) [inline pages/software/building.md:107]: Would recommend switching this order.  Show command for checking the dependencies first, then the command to build the software, then the command to build sw + dependencies.

**Review by dsajdak — CHANGES_REQUESTED** [review]
Just a couple typos and one suggestion.  Otherwise this looks great!

**Comment by aebruno** [comment]
@dsajdak thanks for the feedback. all suggestions have been implemented.

**Review by dsajdak — APPROVED** [review]

**Pull request merged**